.github/workflows/ios_main.yml

```diff
@@ -0,0 +1,25 @@
+name: build
+on: [push]
+
+jobs:
+  build:
+    name: build
+    runs-on: macos-latest
+    strategy:
+        matrix:
+          destination: ['platform=iOS Simulator,OS=13.1,name=iPhone 11']
+          scheme: [OpenDocumentReader]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Checkout submodule
+        run: git submodule sync --recursive && git submodule update --init --recursive
```

Comment:
as far as i know "git submodule sync --recursive" is not necessary? "git submodule update --init --recursive" initializes and updates them anyway

Reply:
you're right, I removed the sync part